fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/impl/FedoraToOCFLObjectIndexImpl.java

```diff
@@ -35,10 +46,36 @@
  */
 @Component
 public class FedoraToOCFLObjectIndexImpl implements FedoraToOCFLObjectIndex {
+
+    private Path indexFilePath = getProperty("fcrepo.ocfl.work.dir") == null ?
```

Comment:
@whikloj :  There are constants these directories in DefaultOCFLObjectSessionFactory.  I would moving DefaultOCFLObjectSessionFactory.STAGING_DIR,  DefaultOCFLObjectSessionFactory.OCFL_STORAGE_ROOT_DIR,  DefaultOCFLObjectSessionFactory.OCFL_WORK_DIR and DefaultOCFLObjectSessionFactory.resolveDir() into OCFLPersistenceConstants and then having FedoraToOCFLObjectIndexImpl use them in order to eliminate the duplicate code.